cfy_manager/components/restservice/restservice.py

```diff
@@ -262,6 +269,22 @@ def _configure_db(self):
                 db.create_amqp_resources(configs)
             else:
                 self._join_cluster(configs)
+            self._fetch_manager_reporter_token()
+
+    @staticmethod
+    def _fetch_manager_reporter_token():
+        sql_stmnt = SELECT_USER_TOKENS_QUERY + " = '{0}'".format(
```

Comment:
you can lose the '+' and just do the format

Reply:
Changed